apps/dev-portal-web/docs/api/cases/invoices/work-with-invoices.ru.mdx

```diff
@@ -0,0 +1,471 @@
+Счет можно создать с указанием данных о плательщике (адрес, телефон, имя) или без.
+В первом случае житель увидит счет в мобильном приложении и оттуда же сможет оплатить.
+Во втором - счет можно будет оплатить только по ссылке.
+Также счет можно напрямую привязать к пользователю. Это удобно делать, например, из мини-приложения.
+В этом случае житель так же увидит счет в мобильном приложении.
+
+## Анонимный счет
+Минимальный набор полей, который в принципе необходим для создания счета, создает анонимный счет.
+Такой счет не виден в мобильном приложении и может быть оплачен только по ссылке.
+**dv** и **sender**
+
+**organization**
```

Comment:
same as above, bold text is not good for structure. Use sub-headings or table

Reply:
Reformatted to list